### PR TITLE
Updates for 1.0.0 release

### PR DESCRIPTION
**Pull request by davidkelly**
* correct urls for downloads
* mention of mac installation
* fixup any mention of `transaction_document`, as it is `transaction_get_result` now
* fixup `log_levels`, no longer plural.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (ingenthr) [inline .gitignore:1]: hi MacOS!

File: modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 2]
```diff
@@ -31,9 +31,18 @@ include::6.5@sdk:shared:partial$acid-transactions.adoc[tag=requirements]
 
 Couchbase transactions require no additional components or services to be configured.
 Simply add the transactions library into your project.
-The latest version, as of October 2020, is 1.0.0.beta.1.
+The latest version is 1.0.0.
```
Comment (ingenthr) [inline modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc]: Would be nice to have the release date or month/year.  We do this elsewhere as it helps people understand how far behind they are.

File: modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 3]
```diff
@@ -43,11 +52,24 @@ $ sudo yum install boost-devel
...
-$ wget https://sdk-snapshots.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
-$ tar xf couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
+$ wget https://packages.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
+$ tar xf couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
 $ sudo yum install couchbase-transactions*.rpm
 ----
 
+=== Installing on Mac OS X
+
+We distribute the Mac libraries through https://brew.sh[homebrew].  To install, once you have homebrew:
```
Comment (ingenthr) [inline modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc]: Mac libraries are available through homebrew.

File: modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc [hunk 4]
```diff
@@ -45,16 +45,25 @@ $ sudo dpkg -i couchbase-transactions*.deb
...
 
 
 === Known Issues
 
-* Owing to https://issues.couchbase.com/browse/MB-41944[MB-41944], this release _should not be used together with a deployment using Sync Gateway_.  
-Owing to this issue there is also a known theoretical data loss problem.  
-This release (as with all beta releases) should not be used in production.  
+* Owing to https://issues.couchbase.com/browse/MB-41944[MB-41944], this release _should not be used together with a deployment using Sync Gateway_.
+Owing to this issue there is also a known theoretical data loss problem.
+This release (as with all beta releases) should not be used in production.
```
Comment (ingenthr) [inline modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc]: Need to remove this line^

**Review by ingenthr — CHANGES_REQUESTED** [review]
-2 mainly for the beta statement, otherwise it looks pretty much okay.

File: modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc (hunk 4 above)
Comment (davidkelly) [reply modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc]: That's in the known issues section for the beta release, not the 1.0.0 known issues.  I think it is ok.

File: modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc (hunk 3 above)
Comment (davidkelly) [reply modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc]: will do

File: modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc (hunk 2 above)
Comment (davidkelly) [reply modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc]: yep why not

File: modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc (hunk 4 above)
Comment (davidkelly) [reply modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc]: But I'm happy to remove it

File: modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 5]
```diff
@@ -43,11 +52,24 @@ $ sudo yum install boost-devel
 
 [source,console]
 ----
-$ wget https://sdk-snapshots.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
-$ tar xf couchbase-transactions-1.0.0-0.beta.1.63.el8.x86_64.tar
+$ wget https://packages.couchbase.com/clients/transactions-cxx/couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
+$ tar xf couchbase-transactions-1.0.0-1.253.el8.x86_64.tar
```
Comment (ingenthr) [inline modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc:56]: I didn't notice this earlier, but I thought we talked about not having that build number in there earlier.  Not going to worry about it for now, but maybe something we can look at later,

**Review by ingenthr — APPROVED** [review]

File: modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc (hunk 4 above)
Comment (ingenthr) [reply modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc]: Ohhh, sorry about that.  I see now.

**Pull request merged**